### DISCUSSION TITLE
Timezone of timestamp!

### DISCUSSION THREAD

**Discussion by tejas-k3** [GrowthVVork/colorflow-habits#20] [Q&A]
Que : What if the project is launched in different timezone? Should we store in DB in UTC ?

**Comment by DipanshuJ** [comment]
MySQL stores Timestamp in UTC , so no need to keep track of UTC expilcitely.
Read this- 
-- [MySQL converts TIMESTAMP values from the current time zone to UTC for storage, and back from UTC to the current time zone for retrieval. (This does not occur for other types such as DATETIME.) ](https://dev.mysql.com/doc/refman/8.0/en/datetime.html#:~:text=MySQL%20converts%20TIMESTAMP%20values%20from%20the%20current%20time%20zone%20to%20UTC%20for%20storage%2C%20and%20back%20from%20UTC%20to%20the%20current%20time%20zone%20for%20retrieval.%20(This%20does%20not%20occur%20for%20other%20types%20such%20as%20DATETIME.))
